src/core_plugins/kibana/public/visualize/editor/sidebar.html

```diff
@@ -14,31 +16,53 @@
         <!-- tabs -->
         <ul class="nav navbar-nav">
           <li ng-class="{active: sidebar.section == 'data'}" ng-show="sidebar.showData">
-            <a class="navbar-link active" ng-click="sidebar.section='data'">Data</a>
+            <a
+              class="vis-editor-subnav-link"
+              g-class="{'is-vis-editor-sub-nav-link-selected': sidebar.section == 'data'}"
+              g-click="sidebar.section='data'"
+            >Data</a>
```

Comment:
Oh wait, I found a couple typos here... on lines 21 and 22, it looks like ng-class and ng-click got turned into g-class and g-click.